scripts/init_sovrin_raet_keep

```diff
@@ -4,7 +4,7 @@ import argparse
 import os.path
 
 from plenum.common.raet import initLocalKeep
-from plenum.common.types import CLIENT_STACK_SUFFIX
+from plenum.common.constants import CLIENT_STACK_SUFFIX
```

Comment:
I agree with you. While doing my changes, I just processed a TODO to rename txn.py to constants.py, because previously it was even worse: we had a module with all constants, but the name of the module was confusing (txn.py contained not only constants related to transactions).
I will create a task to re-factor usage of constants in the code (because I think it's a re-factoring task which is not in the scope of the current story).
In addition to splitting the constants file, I believe we need more well-defined types and enumerations, not just string constants.